benchmark/accuracy.py

```
@@ -0,0 +1,115 @@
+            else:
+                tn += 1
+
+    accuracy  = (tp + tn) / total
+    precision = tp / (tp + fp) if (tp + fp) else 0.0
+    recall    = tp / match_n   if match_n   else 0.0
+    f1        = 2*precision*recall / (precision+recall) if (precision+recall) else 0.0
+    fp_rate   = fp / nomatch_n if nomatch_n else 0.0
+
+    lat_s = sorted(latencies)
+    print(f"\n{'='*60}")
+    print(f"  Accuracy benchmark  —  palavreado")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)